layouts/partials/cards/bis_table.html

```diff
@@ -0,0 +1,41 @@
+{{ $borderClass := "gray-600" }}
+
+{{ if isset . "role"  }}
+  {{ $borderClass = .role }}
+{{ end }}
```

Comment:
Sorry, just caught this. I feel like the code could be a bit shorter if you use the [default](https://gohugo.io/functions/default/) functionality in hugo, so perhaps something like this?

```
{{ $borderClass := default "gray-600" .role }}
```

Or some such.

Reply:
Ah nice, didn't know about this default thing. Works great!

Reply:
Uh, I think the `bis_table.html` template is still missing the `default` statement